src/Middleware/StaticFiles/src/Infrastructure/SharedOptions.cs

```diff
@@ -42,5 +42,10 @@ public PathString RequestPath
         /// The file system used to locate resources
         /// </summary>
         public IFileProvider FileProvider { get; set; }
+
+        /// <summary>
+        /// Indicates whether to add trailing slash at the end of path
```

Comment:
```suggestion
        /// Indicates whether to redirect to add a trailing slash at the end of path. Relative resource links may require this.
```

Reply:
changed

Reply:
Renamed according to your suggestion in next commit